yt/yt/ytlib/api/native/client_queries_impl.cpp

```diff
@@ -327,7 +331,10 @@ std::vector<TString> GetAcosForSubjects(const THashSet<TString>& subjects, const
 
 void VerifyAccessControlObjectExists(const TString& accessControlObject, const IClientPtr& client)
 {
-    auto error = WaitFor(client->NodeExists("//sys/access_control_object_namespaces/queries/" + accessControlObject));
+    auto error = WaitFor(client->NodeExists(Format(
+        "%v/%v",
+        QueriesAcoNamespace,
+        accessControlObject)));
```

Comment:
ToYPathLiteral should be used if `accessControlObject` is user input and may contain ypath-control characters (`/`, `@`, etc)